README.md

```diff
@@ -6,30 +6,40 @@
 
 Port is the Developer Platform meant to supercharge your DevOps and Developers, and allow you to regain control of your environment.
 
-### Docs
+## Documentation
 
-- [Provider Docs](https://registry.terraform.io/providers/port-labs/port/latest/docs)
-- [Port Docs](https://docs.getport.io/)
+---
+
+- [Official Docs](https://registry.terraform.io/providers/port-labs/port/latest/docs)
+- [Port Docs](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/iac/terraform)
 
 ## Installation
 
+---
+
+Terraform utilizes the Terraform Registry to download and install providers. To install port-labs provider, copy and paste the following code into your Terraform file:
+
 ```terraform
 terraform {
   required_providers {
-    port = {
+    port-labs = {
       source  = "port-labs/port-labs"
-      version = "~> 0.4.0"
+      version = "~> 0.10.3"
     }
   }
 }
-provider "port" {}
-
-resource "port-labs_entity" "microservice" {
-  title     = "monolith"
-  blueprint = "microservice_blueprint"
-  properties {
-    name  = "microservice_name"
-    value = "golang_monolith"
-  }
-}
+
+provider "port-labs" {}
 ```
+
+After you have added the code above, run the following command:
+
+```bash
+terraform init
+```
+
```

Comment:
Add here an extra step mentioning getting API credentials and link to our docs - https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/api/#find-your-port-credentials